Source: Based on patch by Ali Polatel, from Gentoo
Upstream: No
Reason: Allow disabling some python modules, to avoid dependencies like bdb, gdbm, tk

From 4c4aa254b82507808e3f6ae218125a175325f635 Mon Sep 17 00:00:00 2001
From: Ingmar Vanhassel <ingmar@exherbo.org>
Date: Tue, 9 Mar 2010 17:09:54 +0100
Subject: [PATCH] Respect PYTHON_DISABLE_MODULES in setup.py

---
 setup.py |    6 +++++-
 1 files changed, 5 insertions(+), 1 deletions(-)

diff --git a/setup.py b/setup.py
index 403b292..79b453a 100644
--- a/setup.py
+++ b/setup.py
@@ -16,7 +16,11 @@ from distutils.command.install import install
 from distutils.command.install_lib import install_lib
 
 # This global variable is used to hold the list of modules to be disabled.
-disabled_module_list = []
+pdm_env = "PYTHON_DISABLE_MODULES"
+if pdm_env in os.environ:
+    disabled_module_list = os.environ[pdm_env].split()
+else:
+    disabled_module_list = []
 
 def add_dir_to_list(dirlist, dir):
     """Add the directory 'dir' to the list 'dirlist' (at the front) if
-- 
1.7.0.2

